mod/deckorate_graphql/lib/graph_q_l/types/company.rb

```diff
@@ -4,6 +4,7 @@ module Types
     class Company < DeckorateCard
       field :headquarters, String, null: true
       field :wikipedia, String, null: true
+      field :official_company_website, String, null: true
```

Comment:
@ethn can we instead use the :wikirate_website card

Reply:
yes!